Homework_2.playground/Contents.swift

```diff
@@ -1,47 +1,113 @@
-class Stack {
-    private let linkedList = LinkedList()
+/// Container for elements organized with LIFO principle.
+class Stack<Element> {
+    private let linkedList = LinkedList<Element>()
     
-    func push(element: Int) {
+    /// Adds a new element on top of the stack
+    ///
+    ///     let stack = Stack<Int>
+    ///     stack.push(element: 7)
+    ///     //head -> 7
```

Comment:
Stack doesn't have a `head` in classic computer science.

Reply:
Also you have arrows (`->`). Is this because you use linked list under-the-hood? Do you think we have to expose it in public API?

Reply:
> Also you have arrows (`->`). Is this because you use linked list under-the-hood? Do you think we have to expose it in public API?

In my case, this is just a visualization of the stack elements without any other meaning.

Reply:
> Stack doesn't have a `head` in classic computer science.

Done.